src/main/webapp/app/entities/ttn/ttn.reducer.ts

```diff
@@ -122,6 +123,14 @@ export const getEntity: ICrudGetAction<ITTN> = id => {
   };
 };
 
+export const getRooms: ICrudGetAction<any> = id => {
```

Comment:
Why `any` here? According to signature of axios.get method it should be `IStorageRoom`